frontend/apps/crates/components/src/input/simple_select/mod.rs

```diff
@@ -3,3 +3,18 @@ mod state;
 
 pub use dom::*;
 pub use state::*;
+
+pub trait Item: Clone {
```

Comment:
I think that the name is a bit too generic, I'd name it something like `SimpleSelectItem` but I don't have super strong feelings about this